### PR TITLE
Bug #414 - PeoplePicker remove button doesn't work - needs to be moved into parent li

### PR DESCRIPTION
**Pull request by manndave**
Bug #414 - PeoplePicker remove button doesn't work - needs to be moved into parent `<li>`

### CODE REVIEW

**Comment by mikewheaton** [comment]
Confirmed, this change to the source order makes the remove buttons work correctly. Everything still displays fine. One more approver and we can merge it in.

File: src/components/PeoplePicker/PeoplePicker.Compact.html [hunk 1]
```diff
@@ -21,8 +21,8 @@
                           <div class="ms-Persona-secondaryText">Sales</div>
                         </div>
                       </div>
-                      <button class="ms-PeoplePicker-resultAction js-resultRemove"><i class="ms-Icon ms-Icon--x"></i></button>
                     </div>
+					<button class="ms-PeoplePicker-resultAction js-resultRemove"><i class="ms-Icon ms-Icon--x"></i></button>
```
Comment (wdo3650) [inline src/components/PeoplePicker/PeoplePicker.Compact.html]: Nit. Remove white space to align button element with its sibling div.

**Comment by manndave** [comment]
@wdo3650: Not sure what's going on.  When I open the sources files (I checked all of them for PeoplePicker) the `button `and sibling `div `elements are all perfectly lined up.

**Comment by wdo3650** [comment]
@manndave Weird, could it be a spaces vs tabs issue?

**Comment by manndave** [comment]
@wdo3650: That might have been it.  I replaced any tabs with spaces, but I'm not sure if it made things better or worse.  It's hard to tell since it all looks right in VSCode either way.

**Comment by mikewheaton** [comment]
@philworthington Are you adding the remove button functionality in the work you're doing on the PeoplePicker?

**Comment by philworthington** [comment]
Yeah all of the functionality had to be rewritten, this includes the remove button functionality.

**Comment by mikewheaton** [comment]
Sorry to close this pull request, but it looks like @philworthington has fixed this issue in a separate branch while working on Fabric 3.0 updates. I believe the structure has changed slightly and this wouldn't merge in.

As always, thanks for your contributions @manndave! We'll get your next PR merged in faster.

**Pull request closed without merging**